src/ducks/pin/PinKeyboard.jsx

```diff
@@ -82,9 +86,9 @@ class PinKeyboard extends React.PureComponent {
               {n}
             </PinButton>
           ))}
-          <PinButton onClick={this.handleRemoveCharacter}>R</PinButton>
+          {this.props.leftButton || <PinButton style={invisible} />}
```

Comment:
Maybe use a class instead of the inline style?

Reply:
I added opacity utility classes in cozy-ui. I will use these classes in a later PR when it will be merged in cozy-ui. Thanks.